scripts/completions/zsh-completions/_wl-mirror

```diff
@@ -26,8 +26,14 @@ _wl_mirror() {
     scalings=("${(@f)$(_wl_mirror_scalings)}")
 
     local -a options
-    options=(
-        '1:output:_values "output" $outputs'
+    if [ ${#outputs} -eq 1 ]; then
+        _message -e "Couldn't find any outputs. Is your XDG_CURRENT_DESKTOP valid?"
+    else
+        options+=('1:output:_values "output" $outputs')
+        options+='--fullscreen-output[set fullscreen target output, implies --fullscreen]:output:_values "output" $outputs'
+    fi
```

Comment:
This fails when you only have one output (e.g. only `eDP-1`). You can check `[[ "${outputs}" == "" ]]` or `[[ -z "${outputs}" ]]` instead, that only matches an empty outputs variable. 

Also, users of desktops that aren't supported here yet _do_ have a "valid" XDG_CURRENT_DESKTOP, but it's just not supported.

My suggestion would be to:

- warn with `"Please set XDG_CURRENT_DESKTOP to enable completion of outputs"` or similar if it is unset
- warn with `"Output completion is unsupported on '$XDG_CURRENT_DESKTOP'"` or similar if the desktop is not sway